swagger/v1/errors.yaml

```diff
@@ -56,10 +56,10 @@ UnsupportedMediaType:
 UnprocessableEntity:
   type: object
   required:
-  - code
-  - detail
-  - source
   - title
+  - detail
+#  - code
+#  - source
```

Comment:
Was this intentional? If so can commented lines be removed perhaps?

Reply:
oops, they should have been deleted - now fixed